src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -323,6 +324,26 @@ public function xdebugAction(): Response
         return new Response($xdebugInfo, 200, ['Content-Type' => 'text/html']);
     }
 
+    /**
+     * Return the JetBrainMono font.
+     *
+     * @throws NotFoundHttpException
+     */
+    public function downloadFontAction(string $font): Response
+    {
+        $this->denyAccessIfProfilerDisabled();
+        $this->profiler?->disable();
+
+        $filename = basename($font).'.woff2';
```

Comment:
Let's be super cautious here and reduce the attack surface by restricting the font families that we can load.

Something like:

```php
if ('JetBrainsMono' !== $fontFamilyName) {
    // return nothing
}
```